Doc/library/calendar.rst

```diff
@@ -516,7 +539,7 @@ The :mod:`calendar` module defines the following exceptions:
 
 .. _calendar-cli:
 
-Command-Line Usage
+Command-line usage
```

Comment:
Do we need this change? 🤔

Reply:
No but this is the preferred style, and updating it matches the page title. It doesn't seem worth its own PR and I'm updating this section anyway in this PR.

https://devguide.python.org/documentation/style-guide/#capitalization